app/scenes/root/root.js

```diff
@@ -41,7 +38,8 @@ export default class Root extends PureComponent {
     componentWillReceiveProps(nextProps) {
         if (this.props.logoutRequest.status === RequestStatus.STARTED &&
             nextProps.logoutRequest.status === RequestStatus.SUCCESS) {
-            this.props.actions.removeStorage();
+
+            // TODO: Purse storage here
```

Comment:
I think this is needed to ensure we remove everything but the server url from the store as is the only way to really logout